debezium-core/src/main/java/io/debezium/transforms/ExtractNewRecordStateConfigDefinition.java

```diff
@@ -109,6 +109,25 @@ public static DeleteHandling parse(String value, String defaultValue) {
             .withWidth(ConfigDef.Width.LONG)
             .withImportance(ConfigDef.Importance.LOW)
             .withDefault("")
-            .withDescription("Adds each field listed from the 'source' element of the payload, prefixed with __ "
```

Comment:
I think this is a left-over from the original work, right? I'll revert that, as the behavior of `add.source.fields` remains unchanged as per our discussion and your update.

Reply:
Indeed, this can be reverted. I looked over this, I guess

Reply:
No worries, I'm applying a few changes (mostly cosmetic) and push a commit for all of it to this PR in a bit.